BossMod/Modules/Dawntrail/Dungeon/D06Alexandria/D063Eliminator.cs

```diff
@@ -140,12 +140,20 @@ public override void AddAIHints(int slot, Actor actor, PartyRolesConfig.Assignme
 
     public override void OnEventCast(Actor caster, ActorCastEvent spell)
     {
-        if ((AID)spell.Action.ID == AID.LightOfSalvationAOE)
+        if ((AID)spell.Action.ID is AID.LightOfSalvationAOE or AID.Subroutine3End or AID.LightOfDevotionAOE)
             CurrentBaits.Clear();
     }
 }
 
-class LightOfDevotion(BossModule module) : Components.SimpleLineStack(module, 3, 40, ActionID.MakeSpell(AID.LightOfDevotionTargetSelect), ActionID.MakeSpell(AID.LightOfDevotionAOE), 5.6f);
+class LightOfDevotion(BossModule module) : Components.SimpleLineStack(module, 3, 40, ActionID.MakeSpell(AID.LightOfDevotionTargetSelect), ActionID.MakeSpell(AID.LightOfDevotionAOE), 5.6f)
+{
+    public override void OnEventCast(Actor caster, ActorCastEvent spell)
+    {
+        if ((AID)spell.Action.ID is AID.Subroutine3End)
+            spell.Action = ActionID.MakeSpell(AID.LightOfDevotionAOE);
```

Comment:
This is scary, this will change the actual class that is stored in replays, that other modules and components will see, etc, will break relogging... I really need to make it readonly.